lib/net/ldap.rb

```diff
@@ -609,13 +609,7 @@ def authenticate(username, password)
   def encryption(args)
     warn "Deprecation warning: please give :encryption option as a Hash to Net::LDAP.new"
     return if args.nil?
```

Comment:
Yeah, I debated whether to include that line in there or not. I'm trying to mimic the exact logic that currently exists, which is "if nil is passed to the encryption method, don't set the encryption". If I omit the line, It will set the encryption to nil.

I'm fine with changing it if you feel it is the desired behavior to have the method set the encryption to nil.